sunpy/map/maputils.py

```diff
@@ -222,11 +238,12 @@ def coordinate_is_on_solar_disk(coordinates):
         Returns `True` if the coordinate is on disk, `False` otherwise.
     """
 
-    if not isinstance(coordinates.frame, Helioprojective):
+    if(_is_coordinates_frame_helioprojective(coordinates)):
+        # Calculate the angle of every pixel from the center of the Sun and compare it the angular
+        # radius of the Sun.
+        return np.sqrt(coordinates.Tx ** 2 + coordinates.Ty ** 2) < solar_angular_radius(coordinates)
```

Comment:
It is getting a PEP8 issue at [Line 244:101](https://github.com/sunpy/sunpy/blob/ec1789c56c04c3db7bfd1b85b5c01766c58568eb/sunpy/map/maputils.py#L244): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (101 > 100 characters)

So shall I declare a new variable `coordinate_angular_distance` and give it the value `np.sqrt(coordinates.Tx ** 2 + coordinates.Ty ** 2)`

```suggestion
        return coordinate_angular_distance < solar_angular_radius(coordinates)
```